whatsagent.env

```diff
@@ -0,0 +1,4 @@
+SECRET_KEY="django-insecure-li%!poi!aywn+197ygv9fc-m14=wp-!$d66ec#32&x6a+mdibn"
+MAILGUN_API_KEY="b56c0f9138110f457f5b8fbf088e037b-c3d1d1eb-f34cfb85"
+GMAIL_SENDER_APP_KEY="rsuivtiotebkdueb"
+EMAIL_SENDER = 'bashobi@gmail.com'
```

Comment:
Should not be here, all your secret keys can't be exposed to Gtihub. 

You need to add `whatsagent.env` to `.gitignore` and make sure this file is not in PR.

Reply:
Added whatsagent.env to gitignore

Reply:
@basroy `whatsagent.env` file is still committed in PR. You need to figure out how to remove it in a way it's not committed as a new file in PR. Adding it to `.gitingore` is not enough if you have already committed the file here.

If we will merge this PR, it will become public and you'll get security breach messages from GitHub about your private keys and everything exposed to the public.

Reply:
The same principle applies to `db.sqlite3`, it's not enough you added a new `.gitignore` entry. `db.sqlite3` has to be removed from PR files. Should not be visible here as a "new project file".

Reply:
Removed by git rm --cached